instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -1024,13 +1028,65 @@ def client_response_hook(send_span, scope, message):
             )
 
 
+def get_distribution_with_fastapi(*args, **kwargs):
+    dist = args[0]
+    if dist == "fastapi~=0.58":
```

Comment:
Do you think it would be easy to get this value from the actual metadata or package.py instead of hardcoding the version here? Not that I expect we are changing this often but still it raised an eyebrow when reviewing :sweat_smile: